helm-chart/values-prod.yaml

```diff
@@ -0,0 +1,23 @@
+# Helm chart values for Production Deployment
+
+djangoDebug: false
+
+replicaCount: 1
+
+resources:
+  requests:
+    cpu: 50m
+    memory: 256Mi
+  limits:
+    cpu: 1000m
+    memory: 1024Mi
+
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx-ingress-public
+  hosts:
+    - host: hermes.lco.global
+      paths:
+        - "/messages"
+        - "/api"
```

Comment:
needs the /submit endpoint

Reply:
I also think paths should not be in dev or prod file. They should be in a single location in the top level values file.